### PR TITLE
Issue #11214: Specified violation message in ModifierOrderCheck

### PR DESCRIPTION
**Pull request by coderdeadpool**
Issue #11214
Specified violation message in ModifierOrderCheck.

### CODE REVIEW

**Comment by coderdeadpool** [comment]
@romani Please review the changes.

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java [hunk 1]
```diff
@@ -103,7 +103,7 @@ public final class InlineConfigParser {
             "com.puppycrawl.tools.checkstyle.checks.regexp.RegexpCheck",
             "com.puppycrawl.tools.checkstyle.checks.whitespace.EmptyForInitializerPadCheck",
             "com.puppycrawl.tools.checkstyle.checks.javadoc.JavadocMethodCheck",
-            "com.puppycrawl.tools.checkstyle.checks.modifier.ModifierOrderCheck",
+            "com.puppycrawl.tools.checkstyle.checks.annotation.MissingDeprecatedCheck",
```
Comment (nrmancuso) [inline src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:106]: @coderdeadpool Why do we not have simple removal of check from this set?

Comment (romani) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:106]: Should be simple removal

**Review by romani — CHANGES_REQUESTED** [review]
See comment above.
All other is good.

File: src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java (hunk 1 above)
Comment (coderdeadpool) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:106]: @nrmancuso @romani I have simply removed this line . However it is showing some conflicts ,  So i rebase the branch and resolved the conflicts.

Comment (nrmancuso) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java:106]: Conflict was not resolved correctly, this line needs to be completely removed.

Comment (coderdeadpool) [reply src/test/java/com/puppycrawl/tools/checkstyle/bdd/InlineConfigParser.java]: Done.

**Review by romani — APPROVED** [review]
Ok to merge as CI pass

**Pull request merged**